code/modules/antagonists/ashwalker/ashwalker.dm

```diff
@@ -0,0 +1,46 @@
+/datum/team/ashwalkers
+	name = "Ashwalkers"
+	var/obj/structure/tendril
+	show_roundend_report = FALSE
+
+/datum/team/ashwalkers/New(starting_members, var/obj/structure/goal)
+	. = ..()
+	tendril = goal
+
+/datum/team/ashwalkers/proc/forge_objectives()
+	var/datum/objective/nest/protect = new
+	protect.team = src
+	protect.update_explanation_text()
+	objectives += protect
+
+/datum/objective/nest
```

Comment:
You could probably make this generic protect_item objective + subtype. Then it's nice for admin abuse. (Optional)